Packs/LogPoint_SIEM_Integration/Integrations/LogPoint_SIEM_Integration/LogPoint_SIEM_Integration.py

```diff
@@ -481,7 +711,7 @@ def fetch_incidents(client, first_fetch, max_fetch):
             last_run = datetime.timestamp(datetime.utcnow() - timedelta(days=1))
     result = client.get_incidents(last_run, now)
     if not result.get('success'):
-        raise DemistoException(result['message'])
+        raise DemistoException(f"ERROR: {result['message']}; last_run: {last_run}; now: {now}")
```

Comment:
```suggestion
        raise DemistoException(f"ERROR: {result.get('message')}; last_run: {last_run}; now: {now}")
```

Reply:
Please note that we have fixed all the similar suggestions in this [216fb4d](https://github.com/demisto/content/pull/12790/commits/216fb4d086c6a9c711a996a0a816d7f6130b6bc5) commit.